src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.books.filter((book) => book.id !== action.payload.id);
+    },
```

Comment:
- [ ] Great work writing this logic here, however, you need to assign the returned array after filtering to the state.books in the `initialState`. Kindly read this [article](https://redux-toolkit.js.org/api/createSlice) to see how actions and slices are used in **_redux toolkit_**. 👍 💻